core-plugins/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,11 @@
+projectId=odf-integration-test
+gcsCsvFile1=testdata/first_name.csv
+gcsCsvFile2=testdata/last_name.csv
+gcsSourceBucket1=
```

Comment:
When will bucket params get filled in? This test will with github actions on PRs, right?

Reply:
It will be filled during the [TestSetupHooks](https://github.com/cdapio/hydrator-plugins/pull/1645/files/e6a093058ecf689af2221190cb2835d13502f8fa#diff-00ff67eba38f39d3a7e4a2c62ef6b2fd6bd9cdc31ecf10a265bf5e426cd13adfR60)

Reply:
got it, thanks!

Reply:
Can we please add a comment?